destination/schema/snowflake/table.go

```diff
@@ -12,21 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package writer
+package snowflake
 
-import (
-	"context"
-	"strings"
+// Table represents a Snowflake table.
+type Table struct {
```

Comment:
`Table` is now used everywhere where we need the schema of a table. It collects the info about the table name, its schema (columns, primary keys) and exposes a way to get "connector columns" (operation, created at, updated at, deleted at).